pkg/datapath/loader/tc.go

```diff
@@ -20,6 +20,10 @@ import (
 // attachSKBProgram attaches prog to device using tcx if available and enabled,
 // or legacy tc as a fallback.
 func attachSKBProgram(device netlink.Link, prog *ebpf.Program, progName, bpffsDir string, parent uint32, tcxEnabled bool) error {
+	if prog == nil {
+		return errors.New("cannot attach a nil program")
```

Comment:
That's true, but doesn't help much figuring out why or what. Maybe `program %s is nil` or something like that?

Reply:
Of course.